website_react/src/App.js

```diff
@@ -0,0 +1,75 @@
+import './App.scss';
+import Form from './components/form/form';
+import Tree from './components/tree/tree';
+
+function App() {
+	const validateFunc = () => {
+        console.log('validate');
+    };
+
+	const data = {
+		fields: [
+			{
+				title: 'Enter Core-Lib name',
+				type: 'string',
+				default_value: 'UserCoreLib',
+				mandatory: true,
+				validatorCallback: validateFunc,
+			},
+			// DB CONNECTION
+			
+
+			// // DB Entity
+			// {
+			// 	title: 'Enter the number of columns',
```

Comment:
add another property called value for showing a form that was already filled before

Reply:
That is correct
`default_value` will  be there if no value exists. 
`value` is used when the user selected a value in advanced